client/src/locales/en.json

```diff
@@ -91,6 +91,10 @@
   "authorization.reset-password.heading": "Reset your password",
   "authorization.reset-password.email-label": "Enter email",
   "authorization.reset-password.button-content": "Reset password",
+  "authorization.actions.no-user-of-email": "There is no user of email {data}.",
```

Comment:
We shouldn't inform about such situation. We should just show information that email was sent and that is all. On backend we should only send if email exists in database. Nobody needs to know if that email is registered in our app or not.